### ISSUE TITLE
Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.

### ISSUE THREAD

**Issue by ppisar** [Sereal/Sereal#77]
When running tests for Sereal-Encoder-3.004,  t/010_desperate.t produces a warning:

```
t/004_testset.t ......................... ok
Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
t/010_desperate.t ....................... ok
```

I'm not sure if this just a false positive, but the test could be more conforming not to emmit warnings.

**Comment by demerphq** [comment]
On 5 January 2015 at 13:51, ppisar <notifications@github.com> wrote:

> When running tests for Sereal-Encoder-3.004, t/010_desperate.t produces a
> warning:
>
> t/004_testset.t ......................... ok
> Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
> Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
> t/010_desperate.t ....................... ok
>
> I'm not sure if this just a false positive, but the test could be more
> conforming not to emmit warnings.
>

Thanks. I was unaware of this issue. I will investigate. What version of
Perl are you using?

Yves


-- 
perl -Mre=debug -e "/just|another|perl|hacker/"

**Comment by ppisar** [comment]
On Mon, Jan 05, 2015 at 05:18:13AM -0800, Yves Orton wrote:
> On 5 January 2015 at 13:51, ppisar <notifications@github.com> wrote:
> 
> > When running tests for Sereal-Encoder-3.004, t/010_desperate.t produces a
> > warning:
> >
> > t/004_testset.t ......................... ok
> > Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
> > Character in 'c' format wrapped in pack at t/lib/Sereal/TestSet.pm line 127.
> > t/010_desperate.t ....................... ok
> >
> > I'm not sure if this just a false positive, but the test could be more
> > conforming not to emmit warnings.
> >
> 
> Thanks. I was unaware of this issue. I will investigate. What version of
> Perl are you using?
> 
Perl 5.20.1. The warning appears on armv7hl platform only.

-- Petr

**Comment by demerphq** [comment]
I just released 3.005 which should fix this. Closing the ticket.

Thanks for the report.
